# Camera
camera_info: "/azure1/rgb/camera_info"
camera_frame: "azure1_rgb_camera_link"
rgb: "/azure1/rgb/image_raw"
rgb_rect: "/azure1/image_rect_color"
depth: "/azure1/depth_to_rgb/image_raw"
point: "/azure1/points2"
width: 1280
height: 720
camera_map_path: /home/demo/catkin_ws/src/assembly_camera_manager/maps/camera/base_to_azure1_rgb_camera_link_20201103-150928.json
K: [609.9959106445312, 0.0, 638.6077270507812, 0.0, 609.9404907226562, 369.51971435546875, 0.0, 0.0, 1.0]
min_depth: 1.0
max_depth: 2.3

# DETR
is_gpu_id: "1"
is_class_names: ["ikea_stefan_side_left", "ikea_stefan_long", "ikea_stefan_middle", "ikea_stefan_bottom"]
# is_weight_path: "/home/demo/catkin_ws/src/assembly_part_recognition/pretrained_weights/furniture_part/DETR_checkpoint0012.pth"
is_weight_path: "/home/demo/catkin_ws/src/assembly_part_recognition/pretrained_weights/furniture_part/DETR_checkpoint0007.pth"
is_thresh: 0.85
seg_thresh: 0.8
roi: [0, 1280, 0, 600]

# MPAAE
pe_gpu_id: "0"
pe_experiment_name: "exp_group/ikea_stefan"
pe_test_config: "mpaae_maskrcnn_azure.cfg"
pe_class_names: ["ikea_stefan_bottom", "ikea_stefan_long", "ikea_stefan_middle", "ikea_stefan_short", "ikea_stefan_side_left", "ikea_stefan_side_right"]
use_mask_crop: [True, False, False, False] # w.r.t is_class names
use_sameWHcrop: [True, True, True, True] 
model_dir: "/home/demo/catkin_ws/src/assembly_knowledge/models"
black_borders: True

# ICP
n_points: 3000
n_iter: 1000
tolerance: 0.005
num_levels: 5

# Reprojection Error (VSD)
cost_type: "step" # step or tlinear
delta: 20
tau: 50

# TCP setup
tcp_ip: 'localhost'
detr_tcp_port: 5000
sim_tcp_port: 5001


